kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/tconstruct/melting.js

```diff
@@ -91,6 +91,41 @@ onEvent('recipes', (event) => {
             temperature: 800,
             time: 58,
             id: 'tconstruct:smeltery/melting/metal/iron/chain'
+        },
+        {
+            ingredient: { item: 'dustrial_decor:gold_chain' },
+            result: { fluid: 'tconstruct:molten_gold', amount: 22 },
+            temperature: 800,
+            time: 58,
+            id: '${id_prefix}molten_gold/chain'

```

Comment:
id's should be descriptive of what they make, and if there's a chance of collision, what the source is.

molten metals are a good example of where things can collide, as many things melt down to them. So I'd prefer to see something like this for the ID's:

\`${id_prefix}molten_gold_from_gold_chain\`

You also need to use backticks \` here for the ${id_prefix} to be used properly.

Reply:
Fixed!